### ISSUE TITLE
[Bug]: "Issues with config" message

### ISSUE THREAD

**Issue by metacortexplus** [LunarVim/LunarVim#1868]
### Problem description

"Issues with config" message on start

### LunarVim version

Idk, how to check?

### Neovim version (>= 0.5)

v0.5.1

### Relevant log output

```shell
NVIM v0.5.1
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wp,-U_FORTIFY_SOURCE -Wp,-D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/builddir/build/BUILD/neovim-0.5.1/x86_64-redhat-linux-gnu/config -I/builddir/build/BUILD/neovim-0.5.1/src -I/usr/include -I/usr/include/luajit-2.1 -I/builddir/build/BUILD/neovim-0.5.1/x86_64-redhat-linux-gnu/src/nvim/auto -I/builddir/build/BUILD/neovim-0.5.1/x86_64-redhat-linux-gnu/include
Compiled by mockbuild@koji

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

### Screenshots

![Screenshot from 2021-10-30 07-57-30](https://user-images.githubusercontent.com/21023720/139520916-576d1f4c-72d8-42a9-a6c4-ea23542edc47.png)

### I have

- [x] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by metacortexplus** [comment]
![Screenshot from 2021-10-30 08-07-39](https://user-images.githubusercontent.com/21023720/139520983-cde1328d-0d65-4032-834e-09ba07d91c83.png)

(idk how to copy text)

**Comment by metacortexplus** [comment]
`:checkhealth`
![Screenshot from 2021-10-30 08-21-42](https://user-images.githubusercontent.com/21023720/139521370-1d95e83d-01bd-4be6-a6aa-07999416d4f1.png)
![Screenshot from 2021-10-30 08-21-30](https://user-images.githubusercontent.com/21023720/139521371-6f0da68e-a5d0-4ac2-ad2f-5a923034a5a6.png)
![Screenshot from 2021-10-30 08-21-12](https://user-images.githubusercontent.com/21023720/139521372-384ef91f-f48e-4bad-9880-a6d6db0596d3.png)
![Screenshot from 2021-10-30 08-21-01](https://user-images.githubusercontent.com/21023720/139521373-7690beb3-0093-49b7-9ba6-c4c682987c39.png)

**Comment by abzcoding** [comment]
please use the `rolling` branch instead of `master`

**Comment by kylo252** [comment]
Should be now fixed after 17648e5a07f8c4fe851b09f3037db58c73fe292f

**Comment by metacortexplus** [comment]
Thank for your generous attention.
